[Build error] Missing string header

When compiling `roctracer` I get the following build error:
```
In file included from /tmp/roctracer-rocm-3.5.0/test/hsa/test/simple_convolution/simple_convolution.h:29,
                 from /tmp/roctracer-rocm-3.5.0/test/hsa/test/simple_convolution/simple_convolution.cpp:23:
/tmp/roctracer-rocm-3.5.0/test/hsa/test/ctrl/test_kernel.h:81:16: error: ‘string’ in namespace ‘std’ does not name a type
   81 |   virtual std::string Name() const = 0;
      |                ^~~~~~
/tmp/roctracer-rocm-3.5.0/test/hsa/test/ctrl/test_kernel.h:28:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
   27 | #include <stdint.h>
  +++ |+#include <string>
   28 | #include <map>
/tmp/roctracer-rocm-3.5.0/test/hsa/test/ctrl/test_kernel.h: In member function ‘bool TestKernel::SetHostDescr(const uint32_t&, const TestKernel::des_id_t&, const uint32_t&)’:
/tmp/roctracer-rocm-3.5.0/test/hsa/test/ctrl/test_kernel.h:105:19: error: ‘malloc’ was not declared in this scope
  105 |       descr.ptr = malloc(size);
      |                   ^~~~~~
/tmp/roctracer-rocm-3.5.0/test/hsa/test/ctrl/test_kernel.h:28:1: note: ‘malloc’ is defined in header ‘<cstdlib>’; did you forget to ‘#include <cstdlib>’?
   27 | #include <stdint.h>
  +++ |+#include <cstdlib>
   28 | #include <map>
In file included from /tmp/roctracer-rocm-3.5.0/test/hsa/test/simple_convolution/simple_convolution.cpp:23:
/tmp/roctracer-rocm-3.5.0/test/hsa/test/simple_convolution/simple_convolution.h: At global scope:
/tmp/roctracer-rocm-3.5.0/test/hsa/test/simple_convolution/simple_convolution.h:50:8: error: ‘string’ in namespace ‘std’ does not name a type
   50 |   std::string Name() const { return std::string("SimpleConvolution"); }
      |        ^~~~~~
/tmp/roctracer-rocm-3.5.0/test/hsa/test/simple_convolution/simple_convolution.h:30:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
   29 | #include "ctrl/test_kernel.h"
  +++ |+#include <string>
   30 | 
```
This issue is similar to https://github.com/ROCm-Developer-Tools/rocprofiler/issues/25
I fixed it by adding
```C++
#include <string>
```
to `test/ctrl/test_kernel.h`

--- a/test/ctrl/test_kernel.h
+++ b/test/ctrl/test_kernel.h
@@ -23,6 +23,7 @@ THE SOFTWARE.
 #ifndef TEST_CTRL_TEST_KERNEL_H_
 #define TEST_CTRL_TEST_KERNEL_H_
 
+#include <string>
 #include <string.h>
 #include <stdint.h>
 #include <map>

